frontend/src/components/user-page/user-page.tsx

```diff
@@ -0,0 +1,42 @@
+import { useAppSelector } from 'hooks/hooks';
+import { RootState } from 'common/types/types';
+import styles from './styles.module.scss';
+
+const UserPage: React.FC = () => {
+  const { user } = useAppSelector(({ auth }: RootState) => ({
+    user: auth.user,
+  }));
+
+  console.warn(user);
+
+  const hasUser = Boolean(user);
+
+  if (!hasUser) {
+    return (
+      <div className={styles.containerUserNotFound}>
+        <h1 className={styles.userNotFound}>Not User Found</h1>
+      </div>
+    );
+  }
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.userInformation}>
+        <img src="" width="256" height="256" />
+        <h3 className={styles.title}>First Name:</h3>
```

Comment:
Are you sure it is semantically correct to call so many tags 'h'?

Reply:
Good catch, H tag is for headings